### DISCUSSION TITLE
ts100 not powering on

### DISCUSSION THREAD

**Discussion by odie2001** [Ralim/IronOS#1550] [General]
My Ts100 works with the usb will go to DFU, take flashes. But when I take off the usb and power on the ts100 it is dead. 
No display and not getting hot. have flashed multi hex and all will take the flash,  
 but it is still dead when powered.

**Comment by Ralim** [comment]
Do you get the `.RDY` file from flashing?
What DFU are you on?
Is this a genuine Miniware TS100?

**Reply by odie2001** [reply]
my DFU 3.45, firmware 2.28 then 2.20. the ts100 installs them and when viewing with the usb on it it shows they are installed.
but when i take off the usb and put power on it. the display is blank and not work. got the firmware from mini site.

**Reply by odie2001** [reply]
https://user-images.githubusercontent.com/124220005/216230599-6e8a1d13-811e-489f-a48f-7a531d0b7997.mov

**Reply by Ralim** [reply]
I'm not quite sure.
The "Ver 2.18" and "CONFIG" are not from this firmware but are the miniware firmware USB modes.
For those you need to use the Miniware forums for support.

**Comment by odie2001** [comment]
thanks. the video was taken with the miniware firmware that was first on it.  just wanted to show what is happening.
I now have v2.0.14daf70 on it, still dead when i put power to it. I think the TS100 has a problem with it's circuit board not the firmware.

**Reply by Ralim** [reply]
Okay if its still dead that sounds like damage to the unit; as I know there is nothing that would stop it working on DC in the code.
